km61/Drapak_Volodymyr/laboratory1.sql

```diff
@@ -1,2 +1,144 @@
 -- LABORATORY WORK 1
 -- BY Drapak_Volodymyr
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+оновлювати дані в таблицях
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER Drapak
+IDENTIFIED BY qwerty123456;
```

Comment:
Need to use structure
 `CREATE USER ....
IDENIFIED BY .....
DEFAULT TABLESPACE ....
TEMPORARY TABLESPACE ... ;`